devito/passes/iet/misc.py

```diff
@@ -204,3 +207,39 @@ def remove_redundant_moddims(iet):
     iet = Transformer(subs, nested=True).visit(iet)
 
     return iet
+
+
+def abridge_dim_names(iet):
+    subs = {}
+    mapper = {}
+    tree = retrieve_iteration_tree(iet)
```

Comment:
this should be called `trees`, not `tree`

it should also be moved at the top of the function since it will be reused by other code blocks later on

so

```
def ...
     trees = ...
     <blank line>
     subs = {}
     ...
```